scopes/react/react/templates/react-provider/files/preview.runtime.ts

```diff
@@ -0,0 +1,42 @@
+import { ComponentContext } from '@teambit/generator';
+
+export const previewRuntimeFile = (context: ComponentContext) => {
+  const { name, namePascalCase: Name } = context;
+
+  return {
+    relativePath: `${name}.preview.runtime.tsx`,
+    content: `import React from 'react';
+import { PreviewRuntime } from '@teambit/preview';
+import { ${Name}, ${Name}Config } from './${name}.aspect';
+import { ReactAspect, ${Name} } from '@teambit/react';
+import { Theme } from '@teambit/base-ui.theme.theme-provider';
```

Comment:
@debs-obrien , we're adding a dependency to a template. this goes against a previous decision.

Reply:
@itaymendel  Can you suggest another way to illustrate how to apply themes as a composition provider?

Reply:
Exactly as I stated in my previous comment we are adding a theme from base ui so need @benjgil to come up with a solution to remove this as the code depends on it so it will break if I remove it.

Reply:
Mocking is what I did in previous templates. This can then just be deleted by consumer easily

Reply:
You mocked an entire component?
We can just comment the theme lines out, so that they're not actually part of the active component but still illustrate how it should be used

Reply:
yes on pair programming session with Ran we mocked a component for the context template to apply a theme with theme context so we wouldn't have to import a theme or component to show how things work. You will see an example by running `bit create react-context theme`

Reply:
No need, just took a look at the code ;) 
That example is different though - here we'd need a full blown component, with separate css and index file and all to illustrate how to create even a basic theme component that can be applied. I think that sort of thing would overwhelm the template